### PR TITLE
Heavy

### PR DESCRIPTION
**Pull request by leviport**

### CODE REVIEW

**Comment by 13r0ck** [comment]
We probably need to look into https://github.com/pop-os/keyboard-configurator/pull/149#issuecomment-1289580415 first

**Review by jackpot51 — APPROVED** [review]

File: keyboards/system76/launch_heavy_1/config.h [hunk 1]
```diff
@@ -0,0 +1,94 @@
+#ifndef CONFIG_H
+#define CONFIG_H
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0x3384
+#define PRODUCT_ID      0x0006
```
[the remark this answers is not in the document]
Comment (leviport) [reply keyboards/system76/launch_heavy_1/config.h]: Incremented, thanks for catching that

**Comment by leviport** [comment]
I did some USB hub benchmarking this morning, and performance is in-line with expectations. I believe this firmware is pretty much ready to go. Am I missing anything?

**Comment by 13r0ck** [comment]
@ids1024 anything missing for the configurator's perspective?

**Comment by ids1024** [comment]
Nothing that I'm aware of, if everything seems to be working well.

**Comment by leviport** [comment]
I found an LED position bug on the tenkey. Some of the LEDs are off a couple positions. I showed Brock and he's working on it now.

**Comment by 13r0ck** [comment]
Fixed!

**Comment by leviport** [comment]
LED persistence problem was fixed by increasing the EEPROM size. I think that's everything

**Pull request merged**